lib/src/api/player/player_api.dart

```diff
@@ -86,4 +86,19 @@ abstract class PlayerApi {
 
   /// Provides access to Analytics related functionality.
   AnalyticsApi get analytics;
+
+  /// Whether casting to a cast-compatible remote device is available.
+  /// `CastAvailableEvent` signals when casting becomes available.
+  Future<bool> get isCastAvailable;
+
+  /// Whether video is currently being casted to a remote device and not played
+  /// locally.
+  Future<bool> get isCasting;
+
+  /// Initiates casting the current video to a cast-compatible remote device.
+  /// The user has to choose to which device it should be sent.
+  Future<void> castVideo();
+
+  /// Stops casting the current video. Has no effect if `isCasting` is false.
```

Comment:
Nit: It actually has an effect when `isCasting` is `false`. It opens the cast dialog, as if `castVideo()` has been called. Maybe we should adapt the doc in this regard.

Reply:
Remove the second sentence: e008c24b9c2dfd8711042529c308760e210e478b